lib/engine/game/g_18_eu/step/modified_dutch_auction.rb

```diff
@@ -258,6 +257,12 @@ def reset_auction(winner, target)
             @auction_triggerer = nil
             next_entity!
           end
+
+          def place_initial_token(minor)
+            hex = @game.hex_by_id(minor.coordinates)
+            city = minor.city.to_i || 0
```

Comment:
wait what, city.to_i when would that be false? if city is nil, to_i will crash

Reply:
That's not new code, just moved, so to be honest I don't remember why it is there. However, it won't crash if nil:

`2.6.3 :001 > a = nil
 => nil 
2.6.3 :002 > a.to_i
 => 0 `

Reply:
If anything it makes the || 0 seem redudant?

Reply:
right

Reply:
so let's remove teh ||0